<launch>
    <node pkg="imu_filter_madgwick" type="imu_filter_node" name="ImuFilter">
        <param name="use_mag" type="bool" value="false" />
        <param name="_publish_tf" type="bool" value="false" />
        <param name="_world_frame" type="string" value="enu" />
        <remap from="/imu/data_raw" to="/sensors/imu"/>
    </node>
    
    <include file="$(find rtabmap_ros)/launch/rtabmap.launch">
        <arg name="args" value="--delete_db_on_start"/>
        <arg name="rgb_topic" value="/sensors/camera/color/image_raw"/>
        <arg name="depth_topic" value="/sensors/camera/aligned_depth_to_color/image_raw"/>
        <arg name="camera_info_topic" value="/sensors/camera/color/camera_info"/>
        <arg name="depth_camera_info_topic" value="/sensors/camera/depth/camera_info"/>
        <arg name="rtabmapviz" value="false"/>
        <arg name="rviz" value="true"/>
    </include>
</launch>
